.storybook/preview.js

```diff
@@ -21,6 +21,7 @@ export const decorators = [
       </WebexDataProvider>
 
       <video id="remote-video" src="./video/ongoing-meeting.mp4" muted autoPlay playsInline loop width="0" height="0" />
+      <video id="remote-share" src="./video/ongoing-share.mp4" muted autoPlay playsInline loop width="0" height="0" />
```

Comment:
this file was taken from https://www.videezy.com/abstract/50576-a-data-screen-of-numbers

just an example, we should replace it with something more screen-sharing like.

Reply:
Is this PR the same as https://github.com/webex/components/pull/208?
I've added a [video here](https://github.com/webex/components/pull/208#pullrequestreview-640554289) that we should use